src/Pattern/Pattern.ts

```diff
@@ -111,15 +85,15 @@ export class Pattern {
    * @param {Object} [options] Options object
    * @param {option.source} [source] the pattern source, eventually empty or a drawable
    */
-  constructor(options: TPatternOptions = {}) {
+  constructor(options: PatternProps = {}) {
```

Comment:
in general i really dislike the word `props` as abbreviation of properties.
Not sure if is because react js popularity or something else, but those are options.

Reply:
renamed to options